modules/serverless-broker-backing-channel-default.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+//  * serverless/admin_guide/serverless-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-broker-backing-channel-default_{context}"]
+= Configuring the default broker backing channel
+
+If you are using a channel-based broker, you can set the default backing channel type for the broker to either `InMemoryChannel` or `KafkaChannel`.
+
+.Prerequisites
+
+* You have administrator permissions on {product-title}.
+* You have installed the {ServerlessOperatorName} and Knative Eventing on your cluster.
+* If you want to use Kafka channels as the default backing channel type, you must also install the `KnativeKafka` CR on your cluster.
+
+.Procedure
+
+* Modify the `KnativeEventing` custom resource (CR) to add configuration details for the `config-br-default-channel` config map:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
```

Comment:
We still have just `apiVersion: operator.knative.dev/v1alpha1` as of Serverless <=1.25